modules/openideal_user/src/EventSubscriber/OpenidealUserEventSubscriber.php

```diff
@@ -52,9 +100,33 @@ public static function getSubscribedEvents() {
     return [
       CKEditorMentionEvent::MENTION_FIRST => 'usersAreMentioned',
       ContentModerationEvents::STATE_CHANGED => 'stateChanged',
+      KernelEvents::RESPONSE => 'response',
     ];
   }
 
+  /**
+   * This method is called when the RESPONSE event is dispatched.
+   *
+   * @param \Symfony\Component\HttpKernel\Event\FilterResponseEvent $event
+   *   Dispatched event.
+   */
+  public function response(FilterResponseEvent $event) {
+    $request = $event->getRequest();
+    if ($request->get('_route') == 'view.frontpage.front_page' && !$request->query->has('tour')) {
```

Comment:
Should we do that only if tour on front page is available?

Reply:
Done